test/gtest/uct/test_sockaddr.cc

```diff
@@ -476,7 +476,7 @@ class test_uct_sockaddr : public uct_test {
         }
     }
 
-    void server_conn_req_recv()
+    void conn_req_recv_cnt_inc()
```

Comment:
IIUC at this point `con_req` is not only recvd but also handeled, I would rename to `m_server_recv_req_cnt` too since its logic was changed

Reply:
but it is currently named as `m_server_recv_req_cnt`

Reply:
do you mean to rename it to `m_server_conn_req_handle_cnt`?

Reply:
new name depends on what this counter means, receiving and handling were in the same cb/function before your change

Reply:
but now they are always in the same function.
the called at the end of the `conn_req_cb`